doc/examples.xml

```diff
@@ -309,3 +309,45 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="BananaTreeGraph">
+<ManSection>
+  <Oper Name="BananaTreeGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers,
+    then this operation returns the <E>banana tree graph</E> 
+    with parameters <A>m</A> and <A>n</A>, as defined below. <P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/BananaTree.html</URL>:
+    <P/>
+
+    <Q>A banana tree graph, as defined by Chen et al. (1997) is a graph obtained by
+    connecting one leaf of each of <A>m</A> copies of an <A>n</A>-star graph with
+    a single root vertex that is distinct from all the stars.</Q>
+    <P/>
+
+    If the optional first argument <A>filt</A> is present, then this should
+    specify the category or representation the digraph being created will
+    belong to. For example, if <A>filt</A> is <Ref Filt="IsMutableDigraph"/>,
+    then the digraph being created will be mutable, if <A>filt</A> is <Ref
+    Filt="IsImmutableDigraph"/>, then the digraph will be immutable. 
+    If the optional first argument <A>filt</A> is not present, then <Ref
+      Filt="IsImmutableDigraph"/> is used by default.<P/>
+
+    See also <Ref Oper="CompleteBipartiteDigraph"/>.
```

Comment:
Could you please move this to before the paragraph about mutability, and also perhaps change it to a reference to the newly-merged `StarDigraph` instead of complete bipartite digraph? I think a reference to `IsUndirectedTree` would be useful too.
```suggestion
    See also <Ref Oper="StarDigraph"/> and <Ref Prop="IsUndirectedTree"/>.
```
If possible, would you mind being more explicit about the correspondence between the abstract definition and the actual vertices/edges that this function gives you, analogous to what I suggested in https://github.com/digraphs/Digraphs/pull/423/files#r592305981

In particular, I think this documentation should note that vertex `1` is that single root vertex, and that the copies of the stars live on the vertices `[2 .. n+1], [n+2 .. 2*n+1], ...` etc, and also which vertex of each star is connected to `1`.